### ISSUE TITLE
Bulk import doesn't work

### ISSUE THREAD

**Issue by JamesArthurHolland** [walterwanderley/sqlc-grpc#16]
Bulk import doesn't work.

```sql
-- name: CreateLocationTransactions :exec
INSERT INTO location_transactions
SELECT
    UNNEST($1::UUID[]) as location_id,
    UNNEST($2::UUID[]) as transaction_id;
```

Produces:

```go
func (s *Service) CreateLocationTransactions(ctx context.Context, req *pb.CreateLocationTransactionsRequest) (*pb.CreateLocationTransactionsResponse, error) {
	var arg CreateLocationTransactionsParams
	arg.Column1 = uuid(req.GetColumn1()) // <--- this is incorrect
	arg.Column2 = uuid(req.GetColumn2()) // <--- so is this

	err := s.querier.CreateLocationTransactions(ctx, arg)
	if err != nil {
		slog.Error("CreateLocationTransactions sql call failed", "error", err)
		return nil, err
	}
	return &pb.CreateLocationTransactionsResponse{}, nil
}
```

Which is wrong because `use of package uuid not in selector`.

It should use uuid.Parse or something instead. How can I fix this? Is this an error with this lib or with protobuf?

**Comment by CodeHariK** [comment]
I added your test case in uuidcheck and it works

```go
func (s *Service) CreateLocationTransactions(ctx context.Context, req *pb.CreateLocationTransactionsRequest) (*pb.CreateLocationTransactionsResponse, error) {
	var arg CreateLocationTransactionsParams
	arg.Column1 = make([]uuid.UUID, len(req.GetColumn1()))
	for i, s := range req.GetColumn1() {
		if v, err := uuid.Parse(s); err != nil {
			err = fmt.Errorf("invalid Column1: %s%w", err.Error(), validation.ErrUserInput)
			return nil, err
		} else {
			arg.Column1[i] = v
		}
	}
	arg.Column2 = make([]uuid.UUID, len(req.GetColumn2()))
	for i, s := range req.GetColumn2() {
		if v, err := uuid.Parse(s); err != nil {
			err = fmt.Errorf("invalid Column2: %s%w", err.Error(), validation.ErrUserInput)
			return nil, err
		} else {
			arg.Column2[i] = v
		}
	}

	err := s.querier.CreateLocationTransactions(ctx, arg)
	if err != nil {
		slog.Error("CreateLocationTransactions sql call failed", "error", err)
		return nil, err
	}
	return &pb.CreateLocationTransactionsResponse{}, nil
}
```

**Comment by walterwanderley** [comment]
Fixed on version v0.19.10
